ictv/pages/utils.py

```diff
@@ -231,3 +273,9 @@ def decorated_function(*args, **kwargs):
 class PermissionGate(object, metaclass=PermissionGateMeta):
     """ Utility class automatically filled by PermissionGateMeta. """
     pass
+
+class DebugEnv(ICTVAuthPage):
```

Comment:
still useful or jut for migration?

Reply:
This is not our code. It was in app.py at line 269 before. But I think Loïc moved it here as it was more coherent